### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by katfang**
@abehaskins Adding UI flows for Auth for Android, iOS, and Web.

### CODE REVIEW

**Comment by katfang** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+* bindings for `ListViews`, `GridViews`, and `RecyclerViews` in Android through custom Firebase Adapters
+* bindings for `UITableViews` and `UICollectionVIews` on iOs through custom Data Sources.
 
 These views are populated with the appropriate `child_added`, `child_changed`, `child_removed`, and `child_moved` events as they occur at the Firebase reference with no additional developer overhead. Additionally, the Adapters and Data Sources can easily be subclassed to add additional functionality  with little additional overhead.
 
 FirebaseUI is built on top of [Firebase](https://www.firebase.com/?utm_source=firebaseui) for realtime data storage and synchronization. [Sign up](https://www.firebase.com/signup?utm_source=firebaseui) for a free account today!
 
 # FirebaseUI Clients
-FirebaseUI is available for both native mobile clients:
+FirebaseUI is available in:
```
Comment (morganchen12) [inline README.md]: Can you change `in` ->`on`? I think it sounds better (FirebaseUI is available on $platform vs FirebaseUI is available in $platform)

File: README.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
 FirebaseUI: Simplyify Mobile Development with Firebase
 ======================================================
 
-FirebaseUI is a library that allows developers to quickly build native mobile applications with Firebase by binding common UI views to Firebase references or queries. Currently FirebaseUI supports `ListViews`, `GridViews`, and `RecyclerViews` on Android through custom Firebase Adapters and `UITableViews` and `UICollectionViews` on iOS through custom Data Sources.
+FirebaseUI is a library that provides common UI elements when building apps. This includes a 
+* basic Auth flow implementation on web, Android, and iOS
+* bindings for `ListViews`, `GridViews`, and `RecyclerViews` in Android through custom Firebase Adapters
+* bindings for `UITableViews` and `UICollectionVIews` on iOs through custom Data Sources.
```
Comment (morganchen12) [inline README.md]: Capitalization:

> ... `UICollectionViews` on iOS through ...

File: README.md [hunk 3]
```diff
@@ -31,13 +39,21 @@ Feature  | Description
...
 [FirebaseRecyclerViewAdapter](https://firebaseui.com/docs/android/index.html) | Binds a Firebase ref or query to a RecyclerView.
+[Firebase Auth UI](https://github.com/firebase/FirebaseUI-Android/tree/master/auth) | UI flows for Authentication
 
 ## iOS
 
 Feature  | Description
 ------------- | -------------
 [UITableViewDataSource](https://firebaseui.com/docs/ios/Classes/FirebaseTableViewDataSource.html)  | Binds a Firebase ref or query to a UITableView.
 [UICollectionViewDataSource](https://firebaseui.com/docs/ios/Classes/FirebaseCollectionViewDataSource.html) | Binds a Firebase ref or query to a UICollectionView.
+[Firebase Auth UI](https://github.com/firebase/FirebaseUI-iOS/tree/master/FirebaseAuthUI) | UI flows for Authentication
```
Comment (morganchen12) [inline README.md:50]: These feature sections are relatively unmaintained, and I can see them falling out of sync with the iOS and Android source repos again in the near future, so I won't worry too much about them in this PR.

**Review by morganchen12 — CHANGES_REQUESTED** [review]
Thanks for doing this work @katfang!

**Comment by katfang** [comment]
@morganchen12 Hmm, I'm not familiar with this change request system ... I attempted to just edit the file and commit changes, but I can't tell if I'm supposed to be able to close out the requests ... but I made the changes. Thanks for the catches :)

File: README.md [hunk 4]
```diff
@@ -1,16 +1,20 @@
 FirebaseUI: Simplyify Mobile Development with Firebase
```
Comment (morganchen12) [inline README.md]: This typo isn't yours, but can you correct it anyway? `Simplyify` -> `Simplify` 😅

Also, FirebaseUI is now beyond mobile, so maybe this whole header should be

> FirebaseUI: Simplifying App Development with Firebase

File: README.md [hunk 5]
```diff
@@ -1,16 +1,20 @@
...
 ======================================================
 
-FirebaseUI is a library that allows developers to quickly build native mobile applications with Firebase by binding common UI views to Firebase references or queries. Currently FirebaseUI supports `ListViews`, `GridViews`, and `RecyclerViews` on Android through custom Firebase Adapters and `UITableViews` and `UICollectionViews` on iOS through custom Data Sources.
+FirebaseUI is a library that provides common UI elements when building apps. This includes a
```
Comment (morganchen12) [inline README.md]: Can you change these lines to 

> FirebaseUI is a library that [...] common UI elements when building apps **with Firebase**. This includes
> - **a** basic Auth flow [...]
> - [...]

File: README.md [hunk 6]
```diff
@@ -1,16 +1,20 @@
 FirebaseUI: Simplyify Mobile Development with Firebase
 ======================================================
 
-FirebaseUI is a library that allows developers to quickly build native mobile applications with Firebase by binding common UI views to Firebase references or queries. Currently FirebaseUI supports `ListViews`, `GridViews`, and `RecyclerViews` on Android through custom Firebase Adapters and `UITableViews` and `UICollectionViews` on iOS through custom Data Sources.
+FirebaseUI is a library that provides common UI elements when building apps. This includes a 
+* basic Auth flow implementation on web, Android, and iOS
+* bindings for `ListViews`, `GridViews`, and `RecyclerViews` in Android through custom Firebase Adapters
+* bindings for `UITableViews` and `UICollectionViews` on iOs through custom Data Sources.
```
Comment (morganchen12) [inline README.md]: Capitalize the S in iOS here.

> ... on **iOS** through custom Data Sources.

**Review by morganchen12 — CHANGES_REQUESTED** [review]

**Comment by katfang** [comment]
@morganchen12 Changes made. :)

File: README.md [hunk 7]
```diff
@@ -1,16 +1,20 @@
-FirebaseUI: Simplyify Mobile Development with Firebase
+FirebaseUI: Simplifying Mobile Development with Firebase
 ======================================================
 
-FirebaseUI is a library that allows developers to quickly build native mobile applications with Firebase by binding common UI views to Firebase references or queries. Currently FirebaseUI supports `ListViews`, `GridViews`, and `RecyclerViews` on Android through custom Firebase Adapters and `UITableViews` and `UICollectionViews` on iOS through custom Data Sources.
+FirebaseUI is a library that provides common UI elements when building apps. This includes a
```
Comment (morganchen12) [inline README.md]: Delete redundant `a` at the end of this line.

> [...] This includes ~~a~~

After this change, feel free to go ahead and merge. The rest looks good.

Comment (katfang) [reply README.md]: Thanks! :)

**Pull request merged**